PennMobile/LaundryPrototype/Networking/LaundryAPIService.swift

```diff
@@ -0,0 +1,56 @@
+
+//
+//  LaundryAPIService.swift
+//  PennMobile
+//
+//  Created by Nathan Aronson on 10/5/25.
+//  Copyright © 2025 PennLabs. All rights reserved.
+//
+
+import Foundation
+import Playgrounds
+
+@MainActor
+class LaundryAPIService {
```

Comment:
```suggestion
enum LaundryAPIService {
```

If you don't plan on instantiating this